src/lib/CollectStore.js

```diff
@@ -106,32 +107,45 @@ export default class CollectStore {
       .sort(sortByName)
   }
 
+  // Get the drive application url using the list of application
+  // FIXME this works only because we need the application list permission for the cozy-bar
+  // and this also supposes that the drive application has the 'drive' slug
+  fetchDriveUrl () {
+    return cozy.client.fetchJSON('GET', '/apps/')
+    .then(body => {
+      const driveapp = body.find(item => item.attributes.slug === 'drive')
+      if (driveapp && driveapp.links) {
+        this.driveUrl = `${driveapp.links.related}/#/files/`
+      }
+    })
+  }
+
   // Fetch all accounts and updates their matching connectors
   fetchAllAccounts () {
-    return accounts.getAllAccounts(cozy.client)
-      .then(accounts => {
-        return Promise.all([accounts, konnectors.getAllErrors(cozy.client)])
+    return Promise.all([
+      accounts.getAllAccounts(cozy.client),
+      konnectors.getAllErrors(cozy.client)
+    ])
+    .then((result) => {
+      const [accounts, errors] = result
```

Comment:
Since the data returned by the Promise.all promise is an array, I don't think so.